samples/user-interface/haptics/src/main/java/com/example/platform/ui/haptics/vibrations/VibrationsRoute.kt

```diff
@@ -39,20 +40,20 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 
 @Composable
-fun HomeRoute(
-    viewModel: HomeViewModel,
+fun VibrationsRoute(
```

Comment:
Could these classes use the Haptic prefix, since the rest of the project is using the term haptic/haptics and not vibration(s)?

This is still a home screen for the haptic samples app, right? Maybe we could rename to HapticSamplesScreen and the resources to haptic_samples_screen_*?

Reply:
The route is not very "home" now since there is a home screen for all the samples in this project. This route is one of the samples along with other routes included in the existing Haptics sample.

The home screen.
![Screenshot_1684209276](https://github.com/android/platform-samples/assets/1237536/a9ddeb96-dbf4-4c19-99c3-002ea8463d46)

Reply:
Got it, thanks for sharing the screenshot!